base_technical_features/models/ir_ui_menu.py

```diff
@@ -0,0 +1,17 @@
+# © 2016 Opener B.V. (<https://opener.am>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class IrUiMenu(models.Model):
+    _inherit = "ir.ui.menu"
+
+    @api.model
+    def _visible_menu_ids(self, debug=False):
+        """Set debug = True, so that group_no_one is not filtered out of the
+        user's groups"""
```

Comment:
Same here

Reply:
That is the docstring available from this commit: [204294](https://github.com/OCA/server-ux/commit/2042944b2127655430035894e2f57d8c6e349ce3)